index.rst

```diff
@@ -109,66 +137,54 @@ syntactically equivalent to a semantic version
 appended.
 
 These are **not** actually semantic versions, in that the compatibility
-guarantees of true semantic versions are not present.  The version
-numbers are constructed purely to aid sortability and version choosing,
-because there are excellent extant tools to manipulate semantic
-versions.
-
-*Within* a given release type for a given image,
-semantic versions (if supported for that type) can be used to compare
-two versions of the Lab container: a higher number is more recent.
-Images are
-comparable *only within a tag type*.  That is, the semantic versions
-from (e.g.) a weekly and a release image are not comparable.
+guarantees of true semantic versions are not present.
+The version numbers are constructed purely to aid sortability and version choosing, because there are excellent extant tools to manipulate semantic versions.
```

Comment:
This part seems to be about the internal implementation strategy in the controller? In that case, I think it would be better to move all the implementation discussion to an appendix, since these details don't really matter for someone making or consuming these versions or looking at the user-visible behavior of Nublado.